### PR TITLE
Fix redirects

### PR DESCRIPTION
**Pull request by XhmikosR**
**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [ ] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) all commits. Pi-hole enforces the [DCO](https://github.com/pi-hole/pi-hole/wiki/Contributing-to-the-project).

---

### CODE REVIEW

File: docs/main/projects.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ last_updated: Sun Sep 06 14:28:14 2020 UTC
 - [Chrome Extension: Pi-Hole List Editor](https://chrome.google.com/webstore/detail/pi-hole-list-editor/hlnoeoejkllgkjbnnnhfolapllcnaglh) ([Source Code](https://github.com/packtloss/pihole-extension))
 - [Splunk: Pi-hole Visualiser](https://splunkbase.splunk.com/app/3023/)
 - [Adblocking with Pi-hole and Ubuntu 14.04 on VirtualBox](https://hbalagtas.blogspot.com/2016/02/adblocking-with-pi-hole-and-ubuntu-1404.html)
-- [Pi-hole stats in your Mac's menu bar](https://getbitbar.com/plugins/Network/pi-hole.1m.py)
+- [Pi-hole stats in your Mac's menu bar](https://github.com/matryer/bitbar-plugins/blob/master/Network/pi-hole.1m.py)
```
Comment (XhmikosR) [inline docs/main/projects.md:14]: Not sure about this; I went with linking to the py file directly.

The current redirect is: `https://getbitbar.com/plugins/Network/pi-hole.1m.py` -> `https://github.com/matryer/bitbar-plugins?from=getbitbar.com&dest=`

File: docs/guides/tor/performance-issues.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ This is not recommended, but here are some things you can do:
 
 ##### Solution 2 - Only use specific Exit Nodes
 
-1. Navigate to [atlas.torproject.org Top Relays](https://atlas.torproject.org/#toprelays).
+1. Navigate to [metrics.torproject.org Top Relays](https://metrics.torproject.org/rs.html#toprelays).
```
Comment (XhmikosR) [inline docs/guides/tor/performance-issues.md:42]: I need a confirmation that the instructions below are still valid since I don't use Tor myself.

**Comment by XhmikosR** [comment]
There is also a 50x error which I'm not sure how to proceed: `https://b3rn3d.herokuapp.com/blog/2014/03/05/tor-country-codes`

File: docs/main/projects.md (hunk 1 above)
Comment (dschaper) [reply docs/main/projects.md:14]: It hasn't been updated in 3 years, go ahead and delete the whole line.

File: docs/guides/tor/performance-issues.md (hunk 2 above)
Comment (dschaper) [reply docs/guides/tor/performance-issues.md:42]: I don't use TOR myself. The changes are okay, if they are wrong someone will helpfully notify us.

**Comment by dschaper** [comment]
That link seems to work for me.

File: docs/main/projects.md (hunk 1 above)
Comment (DL6ER) [reply docs/main/projects.md:14]: Dan is right, we don't even know if this script is still working with the changes we've had in Pi-hole v5.0

**Review by DL6ER — APPROVED** [review]

**Pull request merged**